pkg/channel/external/http_channel.go

```diff
@@ -107,7 +107,7 @@ func (h *HTTPEndpointAppChannel) InvokeMethod(ctx context.Context, req *invokev1
 // invokeMethodV1 constructs the http endpoint request and performs the request while collecting metrics.
 func (h *HTTPEndpointAppChannel) invokeMethodV1(ctx context.Context, req *invokev1.InvokeMethodRequest, appID string) (*invokev1.InvokeMethodResponse, error) {
 	channelReq, err := h.constructRequest(ctx, req, appID)
-	log.Infof("channelReq %v and err %v", channelReq, err)
+	log.Debugf("channelReq %v and err %v", channelReq, err)
```

Comment:
This looks like developer degugging logs, this should be removed.

Reply:
removed :)